packages/protocol/contracts/governance/Validators.sol

```diff
@@ -358,17 +358,7 @@ contract Validators is
    */
   function calculateEpochScore(uint256 uptime) public view returns (uint256) {
     require(uptime <= FixidityLib.fixed1().unwrap(), "Uptime cannot be larger than one");
-    uint256 numerator;
-    uint256 denominator;
-    (numerator, denominator) = fractionMulExp(
-      FixidityLib.fixed1().unwrap(),
-      FixidityLib.fixed1().unwrap(),
-      uptime,
-      FixidityLib.fixed1().unwrap(),
-      validatorScoreParameters.exponent,
-      18
-    );
-    return FixidityLib.newFixedFraction(numerator, denominator).unwrap();
+    return mulExp(FixidityLib.fixed1().unwrap(), uptime, validatorScoreParameters.exponent);
```

Comment:
is this the only place where we use it?

Reply:
There's only one other place we use it, but it's in `StableToken` which has a parameter:
``` 
 /**
   * @return The number of decimal places to which StableToken is divisible.
   */
 uint8 internal decimals_;
```
So unless we want to remove the `decimals_` parameter of StableToken, and thus make all StableToken divisible to the same number of digits as Fixidity fractions, we need to keep the digits argument to `fractionMulExp`.